src/sentry/static/sentry/app/views/releases/list/releaseLanding.tsx

```diff
@@ -2,28 +2,44 @@ import React from 'react';
 import {t} from 'app/locale';
 import styled from 'react-emotion';
 
-import {
-  BashCard,
-  Issues,
-  SuggestedAssignees,
-  Emails,
-  Contributors,
-} from 'sentry-dreamy-components';
+import minified from 'sentry-dreamy-components/dist/minified.svg';
+import emails from 'sentry-dreamy-components/dist/emails.svg';
+import issues from 'sentry-dreamy-components/dist/issues.svg';
+import suggestedAssignees from 'sentry-dreamy-components/dist/suggested-assignees.svg';
+import contributors from 'sentry-dreamy-components/dist/contributors.svg';
+import SentryTypes from 'app/sentryTypes';
 
 import {analytics} from 'app/utils/analytics';
-import SentryTypes from 'app/sentryTypes';
 import withApi from 'app/utils/withApi';
 
 import ReleaseLandingCard from './releaseLandingCard';
 
-const StyledSuggestedAssignees = styled(SuggestedAssignees)`
-  width: 70px;
-  height: 70px;
-  margin-left: 250px;
-
-  @media (max-width: 1150px) {
-    margin-left: 100px;
+type IllustrationProps = {
+  data: string;
+};
+
+class Illustration extends React.Component<IllustrationProps> {
```

Comment:
Can we just make this thing a styled component?

Reply:
maybe. lemme try

Reply:
Something like

```tsx
const illustration = styled({data, className} => (
    <object data={data} className={className}>
        <img src={data} className={className} />
    </object>
))<Props>`
  width: 100%;
  height: 100%;
`;
```

Reply:
tho wouldn't we want some kind of structure since there is a prop? i guess in my mind it has always felt weird to keep use a styled component when new props are involved.

Reply:
but yeah what you wrote it definitely cleaner for sure.

Reply:
ok right because `<props>` can check in the sc now with typescript. neat!